### PR TITLE
`azurerm_cosmosdb_cassandra_datacenter` - expose `seed_node_ip_addresses` property

### PR DESCRIPTION
**Pull request by neil-yechenwei**
Service team and customer is requesting the feature of exposing the property "seed_node_ip_addresses".

image

### CODE REVIEW

File: website/docs/r/cosmosdb_cassandra_datacenter.html.markdown [hunk 1]
```diff
@@ -112,6 +112,8 @@ In addition to the Arguments listed above - the following Attributes are exporte
 
 * `id` - The ID of the Cassandra Datacenter.
 
+* `seed_node_ip_address` - A list of IP Address for the seed nodes in this Cassandra Datacenter.
```
Comment (catriona-m) [inline website/docs/r/cosmosdb_cassandra_datacenter.html.markdown:115]:
```suggestion
* `seed_node_ip_addresses ` - A list of IP Address for the seed nodes in this Cassandra Datacenter. 
```

File: internal/services/cosmos/cosmosdb_cassandra_datacenter_resource.go [hunk 2]
```diff
@@ -109,6 +109,15 @@ func resourceCassandraDatacenter() *pluginsdk.Resource {
 				Optional: true,
 				Default:  true,
 			},
+
+			"seed_node_ip_addresses": {
+				Type:     pluginsdk.TypeList,
+				Computed: true,
+				Elem: &pluginsdk.Schema{
+					Type:         pluginsdk.TypeString,
+					ValidateFunc: validation.IsIPv4Address,
```
Comment (catriona-m) [inline internal/services/cosmos/cosmosdb_cassandra_datacenter_resource.go:118]: since this is computed we don't need to validate this

**Review by catriona-m — CHANGES_REQUESTED** [review]
Thanks @neil-yechenwei I've left a couple of minor comments inline but can take another look once those are addressed. It would also be good if we could add a test case covering this. Thanks!

File: website/docs/r/cosmosdb_cassandra_datacenter.html.markdown (hunk 1 above)
Comment (neil-yechenwei) [reply website/docs/r/cosmosdb_cassandra_datacenter.html.markdown:115]: Updated

File: internal/services/cosmos/cosmosdb_cassandra_datacenter_resource.go (hunk 2 above)
Comment (neil-yechenwei) [reply internal/services/cosmos/cosmosdb_cassandra_datacenter_resource.go:118]: Updated

**Comment by neil-yechenwei** [comment]
@catriona-m , thanks for the comments. I updated PR. Please take another look.

image

**Review by stephybun — APPROVED** [review]
Thanks @neil-yechenwei LGTM 👍

**Pull request merged**